### PR TITLE
Contributor Acknowledgements

### PR DESCRIPTION
**Pull request by triggerNZ**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -39,3 +39,15 @@ Versioning
 The [change log](CHANGELOG.markdown) lists all backwards-incompatible changes to
 the library (i.e. changes which might break existing client code).
 Any such changes require bumping the second number in the version.
+
+Contributors
+------------
+
+- Kristian Domigala [@dkristian](https://github.com/dkristian)
```
Comment (dkristian) [inline README.markdown]: Spelling (Domagala :-) )

Comment (triggerNZ) [inline README.markdown]: Oops, so sorry!!!

**Comment by dkristian** [comment]
:+1:

**Comment by afcowie** [comment]
I originally thought of putting this in a AUTHORS file, but then decided GitHub's contributor metrics were well enough. But happier to see people's names in explicitly.

**Pull request merged**